generated/nitclk/unit_tests/test_nitclk.py

```diff
@@ -1,11 +1,19 @@
 import _matchers
 import _mock_helper
 
+import datetime
 import nitclk
 
 from mock import patch
 
 SESSION_NUM_FOR_TEST = 42
+single_session = [SESSION_NUM_FOR_TEST]
+multiple_sessions = [SESSION_NUM_FOR_TEST, SESSION_NUM_FOR_TEST * 10, SESSION_NUM_FOR_TEST * 100, SESSION_NUM_FOR_TEST + 1]
+
+
+class NitclkSessionTest(object):
```

Comment:
Since it is a test class specific for nitclk, I wanted to keep that in the name.

Docstring added with 'Fake' at the beginning.

Reply:
Adding "Fake" doesn't clarify it, on the contrary. The comment is generic:

"Session objects for drivers that support NI-TClk are expected to have a property of tyep nitclk.SessionReference called tclk.... so this is why we're creating this fake driver class and adding the tclk property."

Maybe rename the class to `NitclkSupportingDriverSession`

> Since it is a test class specific for nitclk, I wanted to keep that in the name.

I don't feel this is necessary since you're inside `test_nitclk.py`.

Reply:
But it is fake and is only for testing.

You don't think something about testing should be in the name?

Anyway, changed

Reply:
Yeah, I suppose you could call it "FakeSessionThatSupportNITClkForTest" or something like that...
Assuming you need the word Test in there, which I don't think is the case.

Reply:
While I think Test should be in there, I didn't feel strongly enough so I had already changed it to what you suggested.